pkg/validations/createcluster/createcluster.go

```diff
@@ -0,0 +1,90 @@
+package createcluster
+
+import (
+	"context"
+	"fmt"
+	"runtime"
+
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/gitops/flux"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/providers"
+	"github.com/aws/eks-anywhere/pkg/providers/validator"
+	"github.com/aws/eks-anywhere/pkg/validations"
+)
+
+type validationManager struct {
```

Comment:
can we make this struct public?

Reply:
Why public? We don't need to access the struct outside of this package